install.sh

```diff
@@ -100,7 +100,7 @@ case $OS in
     bork satisfy satisfy/taps.sh
 
     # need a JDK but can't bork cask this due to sudo
-    brew cask install zulu8
+    brew install --cask zulu8
```

Comment:
Actually, we should change this to zulu11 -- we don't use JDK8 in Spinnaker anymore.

Reply:
updated to 11